src/charm.py

```diff
@@ -799,18 +803,42 @@ def _configure_bessd_for_dpdk(self) -> None:
         if not dpdk.is_configured(container_name=self._bessd_container_name):
             dpdk.configure(container_name=self._bessd_container_name)
 
+    def _are_routes_created(self) -> bool:
```

Comment:
Now that we have validators for the routes being created, should we only create the route conditionally. I.e. in `_configure_and_start_bessd_workload`, we can now have 

```python
if not self._default_route_created():
    self._create_default_route()
if not self._gnb_route_created():
    self._create_ran_route()
```

instead of calling the `ip route replace` command for no reason. 

This also would mean to separate `_routes_are_created` into individual validators.